### DISCUSSION TITLE
astroid_template_zero: Which version is the current one?

### DISCUSSION THREAD

**Discussion by deltapapa01** [templaza/astroid-framework#233] [Q&A] [answered]
Hello,
I still have a small question: I installed astroid template zero on a page, and version 2.5.19 is displayed there, but no update is offered either.
On another page I downloaded and installed astroid template zero freshly, version 2.6.0 is displayed there.
Do I have to update the 2.5.19 manually? So download the fresh packages and then install over them? Or will that be drawn at some point with an update?
Many Thanks,
deltapapa

**Comment by Chacapamac** [comment]
Same question here, I have multiple websites at **Astroid Framework 2.5.20** —> **Template Zero 2.5.19** and Joomla show that the Framework need to be updated (2.6.2), but does not update the Template Zero.
— Framework 2.6.2 come with both Zero/One template version 2..6.2.

**Comment by sonvnn** [comment, marked as answer]
Sorry! I forgot update template xml file on Github. I just updated One and Zero on github.

Thanks & Best Regards,
Sonny

**Reply by Chacapamac** [reply]
Great @sonvnn 
— I can confirm that both templates now appear in the Joomla Update component — To 2.6.2

Good Day!

**Comment by deltapapa01** [comment]
Hi,
confirm too, thank you very much.
is there a changelog for the templates? I only found the changelog for the framework :-)

WBR from GER,
deltapapa
